askbot/media/js/post.js

```diff
@@ -2153,6 +2159,7 @@ EditCommentForm.prototype.createDom = function () {
         var checkBox = this.makeElement('input');
         checkBox.attr('type', 'checkbox');
         checkBox.attr('name', 'suppress_email');
+        checkBox.attr('id', 'suppress_email');
```

Comment:
without id input and label were not connected so click on label didn't change input